src/playwright/transformPlaywrightJson.ts

```diff
@@ -4,7 +4,7 @@ import { toId } from '@storybook/csf';
 
 import { testPrefixer } from './transformPlaywright';
 
-type Story = { id: string; name: string; title: string; parameters?: Record<string, any> };
+type Story = { id: string; name: string; title: string; parameters?: Record<string, any>, kind: string, story: string };
```

Comment:
kind/story are deprecated and shouldn't be used

Reply:
replaced by `story.title` and `story.name`